### DISCUSSION TITLE
Use of image url in src

### DISCUSSION THREAD

**Discussion by ProPhycient5** [vercel/next.js#26262] [Help] [answered]
I wanted to know, how can we use dynamic image in our Image component.
Because we know the below code has solved issue for layout, if our images are in local directory:

But what if I want to use url of the image in src, how can I make it happen ???
Please reply.

```
import Image from 'next/image'
import author from '../public/me.png'

export default function Home() {
  return (
    // When importing the image as the source, you
    // don't need to define `width` and `height`.
    <Image src={author} alt="Picture of the author" />
  )
}
```

**Comment by jacobhq** [comment]
You could do
```js
<Image src="https://example.com/path/to/image.png" alt="Picture of the author" />
```
with no imports apart from `import Image from 'next/image'` needed.

**Reply by ProPhycient5** [reply]
Thanks for the response. But I have tried with your code. It's showing some error. Even I have given hostname in **next.config.js** file. Please help how to resolve that.

```
<Image src="https://i2.wp.com/3.bp.blogspot.com/-4vW18oUZK5o/W5GOdN9GW5I/AAAAAAAAajk/go1e17ybyKMfJY3yvPfQw_7LscATjqc8QCLcBGAs/s1600/victoria%2Bmemorial%252C%2Bkolkata-04.JPG?ssl=1" 
     alt="Picture of the author"
     height={500}
     width={700}  />
```

```
// next.config.js
  module.exports = {
    images: {
      domains: ['i2.wp.com'],
    },
  }
```

**Comment by ProPhycient5** [comment]
Below is error:

Error: Invalid src prop (https://i2.wp.com/3.bp.blogspot.com/-4vW18oUZK5o/W5GOdN9GW5I/AAAAAAAAajk/go1e17ybyKMfJY3yvPfQw_7LscATjqc8QCLcBGAs/s1600/victoria%2Bmemorial%252C%2Bkolkata-04.JPG?ssl=1) on `next/image`, hostname "i2.wp.com" is not configured under images in your `next.config.js`
See more info: https://nextjs.org/docs/messages/next-image-unconfigured-host

**Comment by jacobhq** [comment, marked as answer]
Please take a look at this demo that I have put together:
Demo: https://next-image-testing.vercel.app/
Code: https://github.com/jacobhq/next-image-testing

I used your image and config, and all is fine.

**Comment by ProPhycient5** [comment]
Thanks it worked. Actually after there is no change in code. It worked when given ` yarn dev` after change in code.

Thanks bro for the help. Please do recommend me how to enhance frontend developer skill. Actually I switched and started learning web development/ Software engineering past 3 months ago. Earlier I had background in Computational Physics.
